src/BlobField/index.js

```diff
@@ -14,15 +14,52 @@ export default function BlobField({ collapsed = false }) {
   const blobsRef = useRef(null);
   const history = useHistory();
   const [activeArtist, setActiveArtist] = useState(null);
+  const [blobEvent, setBlobEvent] = useState(null);
   const [height, setHeight] = useState(window.innerHeight);
   const [width, setWidth] = useState(window.innerWidth);
   const [parentWidth, setParentWidth] = useState(window.innerWidth);
   const [parentHeight, setParentHeight] = useState(window.innerHeight);
+  const popoverStyle = usePopooverStyle(blobEvent);
+
+  function usePopooverStyle(event) {
+    let style = {
+      display: 'none'
+    };
+    
+    if (collapsed && event && event.type === "mouseenter") {
+      const isVertical = window.innerWidth > window.innerHeight
+      const count = config.artists.length;
+      const totalLength = isVertical ? window.innerHeight : document.body.clientWidth;
+      const canvasLength = totalLength / count;
+
+      let pos = event.target.position;
+      if (isVertical && pos.x < canvasLength){
+        style = {
+          top: pos.y,
+          right: canvasLength + 5,
+          transform: 'translate(0, -50%)'
+        };
+      }
+      else if (!isVertical && pos.y < canvasLength){
```

Comment:
thoughts on rotating these titles 90° so they line up with the blobs better? Avoids the situation where the name gets cut off at the edge, and IMO it would look more consistent.

Reply:
I thought about it but I don't think it's visual impairment friendly.

Reply:
I suppose that makes sense. And anyway not too many people will see that view, because it's unusual to have a vertical orientation on a device with a mouse.